LibTest/ffi/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ description: co19 tests for Dart FFI library
 publish_to: none
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=3.2.4 <4.0.0'
 
 dependencies:
-  ffi: ^1.0.0
\ No newline at end of file
+  ffi: ^1.0.0
```

Comment:
Nit: might want to bump this to 2.1.0

Reply:
This one still needed?

Reply:
It's best to at least bump it to major version 2. Because otherwise we're pulling in code that no one else is using in the ecosystem.

> This one still needed?

Likely yes if we just `calloc` or `malloc` anywhere in the tests. I believe your IDE should complain if you depend on packages that you're not using anyways.